### DISCUSSION TITLE
Execute the input file in Parallel

### DISCUSSION THREAD

**Discussion by Judy-22** [idaholab/moose#24559] [Q&A Modules: General]
Dears,

My idea is to run an input file in parallel.

I'm finding that for the same input file, the solve converges with one processor but it does not if I increase the number of processors.

**Comment by GiudGiud** [comment]
Hello

The default preconditioning is different in parallel.
You ll want to set your preconditioning parameters manually

Guillaume

**Reply by Judy-22** [reply]
@GiudGiud,

Thanks for your reply.

What do you mean by "You ll want to set your preconditioning parameters manually"? Do I need to select suitable preconditioning parameters?

**Reply by GiudGiud** [reply]
Yes.
Some docs here
https://mooseframework.inl.gov/source/executioners/Steady.html

**Reply by Judy-22** [reply]
Thanks.

I have already read this document to find suitable preconditioning parameters because my problem is fully coupled with 5 coupled variables. With current preconditioning parameters and adaptative time-step, it's possible to go from 0.01 s to 120 s (simulation time 1600s). 
So, do you think that it's possible to find easily adequate preconditioning parameters for parallel execution?

**Reply by GiudGiud** [reply]
For a given physics, your best bet is to study the litterature on how to solve it numerically, then implement what is recommended using petsc_options
